tests/problems.py

```diff
@@ -133,44 +132,127 @@ def nfe(self):
         return self._nfe
 
 
-class Ex4(BaseSDE):
-    noise_type = NOISE_TYPES.general
+class Neural1(BaseSDE):
```

Comment:
Neural1,2,3 doesn't seem be the best names we could give to these classes. Could we use something that's more interpretable w/o reading the impl?